tools/cli/src/index.ts

```diff
@@ -169,28 +169,32 @@ cli
 
 /** Migrator */
 cli
-  .command('migrator')
+  .command('codemod')
   .description('Runs codemod transformations to upgrade LG components')
   .argument(
-    '<migration>',
-    'One of the migrations from: https://github.com/mongodb/leafygreen-ui/blob/main/tools/migrator/README.md#migrations',
+    '<codemod>',
+    'One of the codemods from: https://github.com/mongodb/leafygreen-ui/blob/main/tools/codemods/README.md#codemods-1',
   )
   .argument(
     '<path>',
     'Files or directory to transform. Can be a glob like like src/**.test.js',
   )
-  .option('--dry', 'dry run (no changes are made to files)', false)
   .option(
-    '--print',
+    '--i, --ignore <items...>',
+    'Glob patterns to ignore. E.g. --i **/node_modules/** **/.next/**',
```

Comment:
technically `--ignore="**/node_modules/** **/.next/**"` (with `=` and quotes)

Reply:
That doesn't return the correct array of options
```
--i="one two"
```
```
ignore: [ 'one two' ],
```
but this does
```
--i one two
```
```
ignore: [ 'one', 'two' ],
```
it might be because I'm using `option` and not `argument`?

Reply:
Huh, i was wrong then!